test/Governance.ts

```diff
@@ -544,7 +544,7 @@ describe("Governance test", function () {
         const proposalStateInfo = await this.gov.proposalState(proposalID);
         expect(proposalStateInfo.winnerOptionID).to.be.equal(0n);
         expect(proposalStateInfo.votes).to.be.equal(ethers.parseEther("10.0"));
-        expect(proposalStateInfo.status).to.be.equal(8n);
+        expect(proposalStateInfo.status).to.be.equal(4n);
```

Comment:
Please define javascript constants for the enum value for test readability.
You can even use object for this:
```
const ProposalStatus = {
    INITIAL: 0n,
    RESOLVED: 1n,
    FAILED: 2n,
    CANCELED: 3n,
    EXECUTION_EXPIRED: 4n,
}
...
        expect(proposalStateInfo.status).to.be.equal(ProposalStatus.EXECUTION_EXPIRED);
```